tasks/skylab/StarAlign.wdl

```diff
@@ -130,3 +130,177 @@ task StarAlignFastqPairedEnd {
   }
 
 }
+
+
+task STARsoloFastq {
+  input {
+    Array[File] r1_fastq
+    Array[File] r2_fastq
+    File tar_star_reference
+    File white_list
+    String chemistry
+    String counting_mode
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.9a"
+    Int machine_mem_mb = 64000
+    #ceil((size(tar_star_reference, "Gi")) + 6) * 1100
+    Int cpu = 16
```

Comment:
Do we really need 16 CPU by default ?

Reply:
It will slow it down, but we changed it to 8.